src/Sylius/Bundle/CoreBundle/Form/DataTransformer/TaxonsToCodesTransformer.php

```diff
@@ -61,20 +61,25 @@ public function reverseTransform($value)
             throw new UnexpectedTypeException($value, Collection::class);
         }
 
-        if ($value->isEmpty() || null === $value->get('taxons')) {
+        $taxons = $value->get('taxons');
+
+        // ensure not empty / null
+        if ($value->isEmpty() || null === $taxons) {
```

Comment:
I would say that we don't need this check at all since we're only interested if `$value->get('taxons')` is not null.